Failing test

One of the test I have implemented is failing.
Could you please help me find it.

____________________________ test_get_word_lengths _____________________________
    def test_get_word_lengths():
        text = "Three tomatoes are walking down the street"
>       assert get_word_lengths(text) == [5, 8, 3, 7, 4, 3, 6]
E       assert None == [5, 8, 3, 7, 4, 3, ...]
E        +  where None = get_word_lengths('Three tomatoes are walking down the street')
example.py:29: AssertionError
====================== 1 failed, 1 passed in 0.06 seconds ======================
The command "if [[ $TRAVIS_PYTHON_VERSION == 2.7 ]]; then py.test -vv example.py --cov example; else py.test -vv example.py; fi" exited with 1.

--- a/example.py
+++ b/example.py
@@ -19,8 +19,8 @@ def get_word_lengths(s):
     the word lengths in string s.
     """
     # uncomment next line in step 9
-#   return [len(word) for word in s.split()]
-    return None
+    return [len(word) for word in s.split()]
+    #return None
 
 
 # uncomment this function in step 6

